x/fswap/spec/01_concepts.md

```diff
@@ -0,0 +1,43 @@
+<!--
+order: 1
+-->
+
+# Concepts
+
+## Swap
+
+
+The `x/fswap` module defines a `Swap` type in which a coin is allowed to be swapped into another coin on the chain.
+
+```go
+type Swap struct {
+  FromDenom             string
+  ToDenom               string
+  AmountCapForToDenom   sdk.Int
+  SwapRate              sdk.Dec
+}
+```
+
+## Config
+
+The `x/fswap` module defines a `Config` type for managing the maximum number of Swaps allowed on chain  through `MaxSwaps`. Additionally, `UpdateAllowed` specifies whether `Swap` can be modified.
+
+```go
+type Config struct {
+	MaxSwaps      int
+	UpdateAllowed bool
+}
+```
+
+## Proposal
+
+Typically, a `Swap` is proposed and submitted through foundation via a `MsgSetSwap`.
```

Comment:
`x/foundation` or foundation module. Also, it's just an example of `authority`.
`MsgSetSwap` is NOT a proposal, it's an ordinary message like the others (e.g. x/bank MsgSend). Other modules (x/gov, x/foundation) will include the message in their proposals.

Reply:
> it's just an example of authority.

Could you tell more detail about this? Is it meaning that `MsgSetSwap` could be sent without `authority` or `x/foundation`?
I think it only could be sent by x/foundation not x/gov.

Reply:
As of now, x/gov cannot control x/swap. By design, however, it could be the authority. You may think following enhancements:

1. x/gov message based proposal update
2. removal of x/foundation restrictions on its message based proposals

So, it would be good idea to provide the general concepts first, and giving the current detailed state (it is "concepts" section).